Makefile

```diff
@@ -210,53 +211,67 @@ test.integration: test.integration.dbless test.integration.postgres
 .PHONY: test.integration.enterprise
 test.integration.enterprise: test.integration.enterprise.postgres
 
-.PHONY: test.unit
-test.unit:
-	@go test -v -race \
+.PHONY: _test.unit
+_test.unit:
+	go test -v -race $(GOFLAGS) \
 		-covermode=atomic \
 		-coverpkg=$(PKG_LIST) \
 		-coverprofile=coverage.unit.out \
 		./internal/... \
-		./pkg/...
+		./pkg/... 2>/dev/null
 
-.PHONY: test.integration.dbless
-test.integration.dbless:
+.PHONY: test.unit
+test.unit:
+	@$(MAKE) _test.unit GOFLAGS=$(GOFLAGS)
```

Comment:
here $GOFLAGS does not exist, right?

Reply:
I think this would be more clear if we use `GOFLAGS=""`.

Reply:
But when it's set this way you can add flags from the command line like so:

```
make test.unit GOFLAGS="-count 1337"
```